crates/biome_js_analyze/tests/specs/nursery/useObjectSpread/valid.js

```diff
@@ -0,0 +1,7 @@
+/* should not generate diagnostics */
+({ ...foo });
+({ ...baz, foo: 'bar' });
+Object.assign(foo, { bar: baz });
+Object.assign(foo, bar);
+Object.assign(foo, { bar, baz });
+Object.assign(foo, { ...baz });
```

Comment:
The eslint rule has way more tests, can we add them? https://github.com/eslint/eslint/blob/main/tests/lib/rules/prefer-object-spread.js

Reply:
Added more tests from eslint.